test/e2e-go/e2e_go_cluster_test.go

```diff
@@ -122,12 +124,19 @@ var _ = Describe("operator-sdk", func() {
 				return err
 			}, time.Minute, time.Second).Should(Succeed())
 
+			By("ensuring the created resource object gets reconciled in controller")
+			managerContainerLogs := func() string {
+				logOutput, err := tc.Kubectl.Logs(controllerPodName, "-c", "manager")
+				Expect(err).NotTo(HaveOccurred())
+				return logOutput
+			}
+			Eventually(managerContainerLogs, time.Minute, time.Second).Should(ContainSubstring("Successfully Reconciled"))
```

Comment:
I removed this because this log line was removed from the v0.7.0 controller-runtime reconciler handler. The best way to detect whether a reconcile occurred is through [metrics](https://github.com/operator-framework/operator-sdk/pull/4056/files#diff-a39c326959604134a7985f6701488e7afd31bb2afb482e3315a5e4855172ce67R190), since this will break for go/v3 code when we start testing it.
```suggestion
```